lib/Runtime/ByteCode/Scope.h

```diff
@@ -265,6 +267,9 @@ class Scope
     void SetIsBlockInLoop(bool is = true) { isBlockInLoop = is; }
     bool IsBlockInLoop() const { return isBlockInLoop; }
 
+    void SetContainsWith(bool does = true) { containsWith = does; }
```

Comment:
No, we can't. The attribute means that there's a with scope nested somewhere within the current scope.